packages/app/src/runner/SpecRunnerHeaderOpenMode.vue

```diff
@@ -184,6 +182,12 @@ watchEffect(() => {
 
 const autIframe = props.getAutIframe()
 
+const inputZIndex = computed(() => {
+  // input needs to be above the Studio prompt overlay
+  // but other times it needs to be below other resizable panels
+  return studioStore.needsUrl ? 51 : 5
```

Comment:
is the studio prompt overlay have a z-index of 50 or something near that? Is there a better way to determine the z-index value here over static values?